jobs/build/build-sync/Jenkinsfile

```
@@ -78,7 +78,7 @@ node {
 
     stage("Version dumps") {
         buildlib.doozer "--version"
-        echo "${buildlib.DOOZER_BIN}"
+        sh "which doozer"
```

prefer `command -v` see https://stackoverflow.com/a/677212
```suggestion
        sh "command -v doozer"
```